packages/kbn-es/src/serverless_resources/security_roles.json

```diff
@@ -219,47 +331,88 @@
             "logs-*",
             "packetbeat-*",
             "winlogbeat-*",
-            ".lists*",
-            ".items*",
             ".asset-criticality.asset-criticality-*"
           ],
-          "privileges": ["read", "write"]
+          "privileges": [
+            "read",
+            "write"
+          ]
         },
         {
           "names": [
             ".alerts-security*",
             ".preview.alerts-security*",
-            ".internal.preview.alerts-security*",
+            ".internal.preview.alerts-security*"
+          ],
+          "privileges": [
+            "read",
+            "write",
+            "manage"
+          ]
+        },
+        {
+          "names": [
+            ".lists*",
+            ".items*",
             ".siem-signals-*"
           ],
-          "privileges": ["read", "write", "manage"]
+          "privileges": [
+            "read",
+            "write",
+            "manage",
+            "view_index_metadata"
+          ]
```

Comment:
This is the only change here for `soc_manager` where `siem-signals-*`,`.lists*` and `.items` get privileges for [enabling detections](https://www.elastic.co/guide/en/security/master/detections-permissions-section.html). The rest is eslint changes